data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -1,9 +1,19 @@
 {
-  "id": "ARK/URI(?)",
-  "label": "",
+  "id": "URI",
   "type": "CV",
   "place_type": "CV",
   "uniform_name": "",
+  "alt_name": [
+    "array of language-mapped strings"
+  ],
+  "role": [
```

Comment:
I don't like 'role' necessarily for places. But some kind of mechanism similar to the role field in persons is needed so we can gather the attestations in SDP Objects. We would have name variants from these attestations, and we can also have role-like information ("place of copying", "previous repository", "current repository", "location of donation", etc.)